.github/workflows/test.yaml

```diff
@@ -371,7 +386,7 @@ jobs:
           echo "::add-matcher::.github/spread-problem-matcher.json"
           export NESTED_BUILD_SNAPD_FROM_CURRENT=true
           export NESTED_ENABLE_KVM=true
-          spread -abend google-nested:${{ matrix.system }}:tests/nested/...
+          spread -abend google-nested:${{ matrix.system }}:tests/nested/... | tee spread.log
```

Comment:
this again broke spread tests reporting failing spread tests in GitHub Actions as being successful